src/configuration/action.rs

```diff
@@ -47,13 +49,28 @@ impl Action {
 
         // iterate over data items to allow any data item to skip the entire descriptor
         for data in self.data.iter() {
-            match &data.item {
+            let (key, value) = match &data.item {
                 DataType::Static(static_item) => {
-                    let mut descriptor_entry = RateLimitDescriptor_Entry::new();
-                    descriptor_entry.set_key(static_item.key.to_owned());
-                    descriptor_entry.set_value(static_item.value.to_owned());
-                    entries.push(descriptor_entry);
+                    (static_item.key.to_owned(), static_item.value.to_owned())
                 }
+                DataType::Expression(cel) => (
+                    cel.key.clone(),
+                    match cel
+                        .compiled
+                        .get()
+                        .expect("Expression must be compiled by now")
+                        .eval()
+                    {
```

Comment:
This is how we've send data to limitador so far, losing the type information... just a string: `true`, `42`, `bob`